# HG changeset patch
# User Andreas Weber <andreas.weber@hs-offenburg.de>
# Date 1378899105 -7200
#      Wed Sep 11 13:31:45 2013 +0200
# Node ID 266fafd1ffc1187773cb8b86c4f9a81144b2d75c
# Parent  a5de727e3795d31d68f194652c7496fe6c92ba27
fix 2 strcmp<->strncmp issues from 3f0ed69d21c6

* __go_draw_axes__.m: replaced strncmp with strcmp
* plot/uigetfile.m: removed length specifier from strcmpi

diff --git a/scripts/plot/private/__go_draw_axes__.m b/scripts/plot/private/__go_draw_axes__.m
--- a/scripts/plot/private/__go_draw_axes__.m
+++ b/scripts/plot/private/__go_draw_axes__.m
@@ -638,17 +638,17 @@ function __go_draw_axes__ (h, plot_strea
              zcol = obj.zdata(:,i);
            else
              zcol = zeros (size (xcol));
            endif
          endif
 
          if (! isnan (xcol) && ! isnan (ycol))
            ## Is the patch closed or not
-           if (strncmp (obj.facecolor, "none"))
+           if (strcmp (obj.facecolor, "none"))
              hidden_removal = false;
            else
 
              if (isnan (hidden_removal))
                hidden_removal = true;
              endif
              if (nd == 3)
                if (numel (xcol) > 3)
diff --git a/scripts/plot/uigetfile.m b/scripts/plot/uigetfile.m
--- a/scripts/plot/uigetfile.m
+++ b/scripts/plot/uigetfile.m
@@ -169,17 +169,17 @@ function [retfile, retpath, retindex] = 
       prop = varargin{i};
       val = varargin{i + 1};
       if (strcmpi (prop, "position"))
         if (ismatrix (val) && length (val) == 2)
           outargs{4} = val;
         else
           error ("uigetfile: expecting 2-element vector for position argument");
         endif
-      elseif (strcmpi (prop, "multiselect", 11))
+      elseif (strcmpi (prop, "multiselect"))
         if (ischar (val))
           outargs{5} = tolower (val);
         else
           error ("uigetfile: expecting string argument (on/off) for multiselect");
         endif
       else
         error ("uigetfile: unknown argument");
       endif
